host/libhackrf/src/hackrf.c

```diff
@@ -269,6 +275,9 @@ static int free_transfers(hackrf_device* device)
 		free(device->transfers);
 		device->transfers = NULL;
 	}
+
+	libusb_free_transfer(device->flush_transfer);
```

Comment:
Should this be conditional to avoid a double free?

Reply:
It doesn't need to be, the function is [documented](https://libusb.sourceforge.io/api-1.0/group__libusb__asyncio.html#ga6ab8b2cff4de9091298a06b2f4b86cd6) to be a no-op with a null pointer.

Reply:
(also note that I set the `LIBUSB_TRANSFER_FREE_BUFFER` flag on that transfer, so this call will free the 32KB buffer as well)